core/schema/module.go

```diff
@@ -1175,6 +1175,7 @@ func (s *moduleSchema) updateCodegenAndRuntime(
 			if bytes.Contains(gitIgnoreContents, []byte(fileName)) {
 				continue
 			}
+			fileName := strings.TrimPrefix(fileName, "/")
```

Comment:
I'm not sure about this. If you have `/sdk` it may mean you want to exclude the `sdk` dir next to `.gitignore`, but not `client/sdk`.

Reply:
See the line after - we always unconditionally prefix the `/`. This PR just ensures we don't end up with double `//` if the SDK already included one.